addons/lunch/models/lunch_supplier.py

```diff
@@ -230,7 +230,7 @@ def _get_current_orders(self, state='ordered'):
             ('supplier_id', 'in', available_today.ids),
             ('state', '=', state),
             ('date', '=', fields.Date.context_today(self.with_context(tz=self.tz))),
-        ], order="user_id, name")
+        ], order="user_id, product_id")
```

Comment:
This changes the ORDER BY clause by adding a LEFT JOIN to the product table, and using the product's model order.  But in the case of lunch products, I don't think it will cause a performance issue.